projects/packages/forms/src/blocks/contact-form/variations.js

```diff
@@ -26,7 +26,7 @@ const variations = [
 			variationName: 'default-empty',
 		},
 		scope: [ 'transform' ],
-		isActive: ( { variationName } ) => variationName !== 'multistep',
+		isActive: ( { variationName } ) => [ 'multistep', 'horizontal' ].indexOf( variationName ) < 0,
```

Comment:
```suggestion
		isActive: ( { variationName } ) => ![ 'multistep', 'horizontal' ].includes( variationName ),
```

Tiny tiny nit, `.includes` is more readable.